lib/client/mfa.go

```diff
@@ -126,6 +126,7 @@ func promptU2FChallenges(ctx context.Context, proxyAddr string, challenges []*pr
 		})
 	}
 
+	log.Debugf("prompting U2F devices with facet %q", facet)
```

Comment:
Feels like a debug leftover, need this?

Reply:
This is left on purpose.
If a user mis-configures U2F facets (proxy addresses, basically) on the `auth_service`, it's really hard to debug when they use a different value in `tsh --proxy=...` and U2F auth will fail.